### PR TITLE
Make gde pip installable

### LINKED ISSUE #3: Make library pip-installable directly from source
Make it possible to pip install a copy of `graph_def_editor` from the source code repository directly into a local virtualenv. This work does *not* include generating a pip-installable tarball or posting such a tarball on PyPi.

### PR DESCRIPTION
**Pull request by BryanCutler**
Created setup.py to build distribution. 

Use command `python setup.py sdist bdist_wheel` to generate a python wheel in `dist/graph_def_editor*.whl`.

This can then be installed with `pip install dist/graph_def_editor*.whl`.

Fixes #3

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        'Intended Audience :: Developers',
+        'Intended Audience :: Education',
+        'Intended Audience :: Science/Research',
+        'Topic :: Scientific/Engineering :: Mathematics',
+        'Topic :: Software Development :: Libraries :: Python Modules',
+        'Topic :: Software Development :: Libraries',
+    ],
+    license='Apache License, Version 2.0',
+    maintainer='Graph Def Developers',
+    maintainer_email='',
```
Comment (BryanCutler) [inline setup.py]: @frreiss take a look at some of the above fields.  I tried to make my best guess for some, but not sure what to put for maintainer/email..

Comment (frreiss) [reply setup.py]: Looks good to me.

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
+# Copyright 2018 IBM. All Rights Reserved.
+# Copyright 2015 The TensorFlow Authors. All Rights Reserved.
```
Comment (frreiss) [inline setup.py]: You can drop this second copyright notice -- no Google IP here.

Comment (BryanCutler) [reply setup.py]: Oh, ok - I copied it from one of the source files

File: setup.py (hunk 1 above)
Comment (BryanCutler) [reply setup.py]: is there anything I can put for the email?

**Pull request merged**